menu-pages/schoolAid-login.html

```diff
@@ -107,6 +108,10 @@
             </li>
         </ul>
     </nav>
+    <span class="openbtn" onclick="openNav()">&#9776; Меню</span>
+
+
+

```

Comment:
same here..look at the menu-pages/schoolAid-course.html . I changed it to give you a guide